index.html

```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
       </div>
       <div class="col-7_md-12_md-first hero">
         <div>
-          <img class="img md-hidden" src="{{ "/images/hero.svg" | relative_url }}" />
-          <img class="img md-gt-hidden" src="{{ "/images/hero-mobile.svg" | relative_url }}" />
+          <img class="img" srcset="./images/hero.png 1x, ./images/hero@2x.png 2x, ./images/hero@3x.png 3x" alt="Crossplane hero image" />
         </div>
       </div>
     </div>
   </div>
 </div>
 
-<div id="kinds">
+<div id="use-cases">
   <div>
-    <div class="grid-center">
-      <div class="col-10_sm-12">
-        <h2>The many flavors of managed services we support</h2>
+    <div class="grid">
+      <div class="col-6_md-12 img">
+        <img srcset="./images/feature-1.png 1x, ./images/feature-1@2x.png 2x, ./images/feature-1@3x.png 3x" alt="Provision managed services" />
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper